### PR TITLE
chore: build.sh script unnecessarily requires lerna to be globally installed

### PR DESCRIPTION
**Pull request by timmattison**
Either lerna needs to be installed as a dependency or run through npx. Otherwise build.sh fails in the middle of a build on a system without it installed.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by timmattison** [comment]
Exemption Request

My changes un-break the build. I'm not sure how I would write a test for it.

**Review by madeline-k — APPROVED** [review]
Thanks for the PR @timmattison! Changing this to a "chore" instead of a "fix" because it's developer/contributor facing and not customer facing. (Anything labeled as "fix" or "feat" ends up in the changelog and release notes, which is not necessary here.)

**Review by mrgrain — CHANGES_REQUESTED** [review]
@timmattison @madeline-k  Are you sure that's what's going on? Because in line 38 the scripts adds `node_modules/.bin` to the path and in line 48 it does a `yarn install` which would install lerna at the right version.

And this does work on my machine (I know, I know) and in CI. So something else must be going on.

Either way, if we want to use `npx` it would at least have to be qualified to the correct version `npx lerna@^4.0.0`


PS: @madeline-k feel free to just dismiss my review if I'm blocking something =)

**Comment by madeline-k** [comment]
> @timmattison @madeline-k Are you sure that's what's going on? Because in line 38 the scripts adds `node_modules/.bin` to the path and in line 48 it does a `yarn install` which would install lerna at the right version.
> 
> And this does work on my machine (I know, I know) and in CI. So something else must be going on.
> 
> Either way, if we want to use `npx` it would at least have to be qualified to the correct version `npx lerna@^4.0.0`
> 
> PS: @madeline-k feel free to just dismiss my review if I'm blocking something =)

@mrgrain You are completely correct, the script does install `lerna`. And the goal of this change is to actually use the version of `lerna` installed locally in the projects dependencies. The problem now is that running the `lerna` command will run whatever version is installed globally, if there is one. I think the original title of the PR was a bit misleading. The `lerna` dependency is not missing, we just need the script to execute it from the projects dependencies. 

(The `npx` command will run requested packages from the local project dependencies first, before trying to get it from the npm cache or the internet. See [docs](https://docs.npmjs.com/cli/v7/commands/npx).)

**Review by madeline-k — APPROVED** [review]

**Comment by madeline-k** [comment]
@mrgrain, dismissing your review. We can always revert this if I am incorrect on something, but I think the change is good as is.

**Review by TheRealAmazonKendra — APPROVED** [review]
Looks like our automations weren't working correctly here when previously approved.

**Review by TheRealAmazonKendra — DISMISSED** [review]
Actually, it looks like a piece of the command got removed with this change. We want the concurrency setting left in there.

File: build.sh [hunk 1]
```diff
@@ -83,7 +83,7 @@ concurrency=$(node -p 'Math.max(1, require("os").cpus().length - 1)')
 
 echo "============================================================================================="
 echo "building..."
-time lerna run $bail --stream --concurrency=$concurrency $runtarget || fail
+time npx lerna run $bail --stream $runtarget || fail
```
Comment (madeline-k) [inline build.sh:86]: Add back the concurrency flag (which was added after the original authoring of this PR):
```suggestion
time npx lerna run $bail --stream --concurrency=$concurrency $runtarget || fail
```

Thanks @TheRealAmazonKendra for catching that.

**Review by madeline-k — CHANGES_REQUESTED** [review]

**Review by madeline-k — DISMISSED** [review]

**Review by TheRealAmazonKendra — APPROVED** [review]

**Pull request merged**